cassandra/cqlengine/functions.py

```diff
@@ -12,20 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from __future__ import division
```

Comment:
Per PEP-238 the effect of this future import in 2.x was to enforce true division "throughout a module".  In Python3 / is understood to mean true division generally... so there is no impact to this removal t'all.